container/single-instance/Dockerfile

```diff
@@ -2,7 +2,7 @@
 #!BuildTag: openqa-single-instance:latest opensuse/openqa-single-instance:latest opensuse/openqa-single-instance:%PKG_VERSION% opensuse/openqa-single-instance:%PKG_VERSION%.%RELEASE%
 
 # hadolint ignore=DL3006
-FROM opensuse/tumbleweed
+FROM opensuse/leap:15.5
```

Comment:
We already upgraded the other containers to 15.6 . I would prefer Tumbleweed as we wouldn't need to care about upgrading those versions but if there are specific problems on Tumbleweed right now then please upgrade to 15.6 and state here why we can't use Tumbleweed

Reply:
1. Tumbleweed doesn't work with Github codespaces.
That reminds me that I wanted to create a bugreport for this.
Try it out in this example repo: https://github.com/perlpunk/test-codespace-opensuse-tumbleweed
See also https://progress.opensuse.org/issues/87695#note-18

2. 15.6 doesn't work because of other problems

> I'm now trying out Leap 1.5, because 15.6 did not work:
> During building in OBS, it has a newer version of qemu available than later when the container is running. This seems to be special to Leap 15.6.

https://progress.opensuse.org/issues/87695#note-22

Try it out yourself; build the single instance container in a branch for 15.6 and then run it and run openqa-bootstrap on it. The zypper install will run into a conflict.